api/provision/sample-fn.ts

```diff
@@ -0,0 +1,14 @@
+import { Context } from "aws-lambda";
+
+/**
+ * Sample lambda used as a placeholder prior to implementation of other lambdas
+ *
+ * @param stateInput - input to the state task that is processed
+ */
+export async function handler(stateInput: any, context?: Context): Promise<unknown> {
```

Comment:
_eventually_ (i dont think we want to do it now), we are going to want to make this more specific than an `any` not necessarily in the function definition but for sure towards the top of the body.

Reply:
Agreed - that gets us closer to a contract-based approach for making these functions more modular and tightly unit testable.

Reply:
Yup, that sounds good to me. This was just a placeholder function for the time being.